start.ps1

```diff
@@ -75,8 +75,8 @@ function install {
 
 function set_env {
     # set all env vars in .env file
-    if (-not (Test-Path "$($repo_root)\.env")) {
```

Comment:
I am not sure about this change, but in my tests the way I am duing it was working.

Reply:
hm weird, i thought i'd adressed that already. nevermind, i'll check this again at home